feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -789,24 +806,55 @@ class BillingWrapper(
 
     private fun BillingClient.queryProductDetailsAsyncEnsuringOneResponse(
         @BillingClient.ProductType productType: String,
-        params: QueryProductDetailsParams,
-        listener: ProductDetailsResponseListener,
+        nonEmptyProductIds: Set<String>,
+        listener: (result: BillingResult, products: List<GoogleProductData>) -> Unit,
     ) {
+        val canUseProductDetails = isProductDetailsUnupported()
```

Comment:
Should this be negated?

Reply:
I'd maybe add an extra comment here, explaining _why_ we need to do this and pointing to the GH issue and PR. It'd otherwise be hard for a maintainer to understand when the product details would be unsupported on BC5